airootfs/etc/skel/.config/hypr/keybinds.conf

```diff
@@ -0,0 +1,133 @@
+# See https://wiki.hyprland.org/Configuring/Keywords/ for more
```

Comment:
Kindly add [this](https://wiki.hyprland.org/Configuring/Keywords/#sourcing-multi-file) to the conf files. Append `source = ~/.config/hypr/keybinds.conf` to the `hyprland.conf` file.

Reply:
Done :saluting_face: